mypy/constraints.py

```diff
@@ -702,7 +753,7 @@ def visit_tuple_type(self, template: TupleType) -> List[Constraint]:
             isinstance(actual, Instance)
             and actual.type.fullname == "builtins.tuple"
         )
-        unpack_index = find_unpack_in_tuple(template)
+        unpack_index = find_unpack_in_list(template.items)
```

Comment:
I wanted to migrate this to use the other helpers, but couldn't figure out how to cleanly do it.